blocks/test/full-content.js

```diff
@@ -165,7 +165,9 @@ describe( 'full post content fixture', () => {
 				) );
 			}
 
-			const serializedActual = serialize( blocksActual );
+			// `serialize` doesn't have a trailing newline, but the fixture
+			// files should.
+			const serializedActual = serialize( blocksActual ) + '\n';
```

Comment:
Should we trim here in case `serialize` changes in the future to add its own trailing newline or... update this logic if/when that time comes?